modules/i3s/docs/api-reference/i3s-loader.md

```diff
@@ -17,6 +24,16 @@ A loader for loading an [Indexed 3d Scene (I3S) layer](https://github.com/Esri/i
 | Data Format    | [Data formats](#data-formats)                       |
 | Supported APIs | `load`, `parse`                                     |
 
+## I3S format versions support
+- `I3S 1.6`
+- `I3S 1.7`
+  - Node pages
+  - Draco compression
+  - PBR materials
+  - Compressed textures (.dds format)
```

Comment:
dds is a container, we should specify what texture compressions we support.

Reply:
https://github.com/Esri/i3s-spec/blob/master/docs/1.7/textureSetDefinitionFormat.cmn.md
> Only DXT1 (no alpha) and DXT5 (alpha channel) are supported.